.github/workflows/land-blocking.yml

```diff
@@ -89,6 +89,19 @@ jobs:
               // Post comment on PR then fail this workflow
               should_fail = true;
             }
+            // Add repro cmd to message
+            try {
+              let env_vars = process.env;
+              body += "\nRepro cmd:\n";
+              body += `
+                $(aws ecr get-login --no-include-email --region us-west-2)
+                ./scripts/cti --tag dev_${env_vars.LIBRA_GIT_REV} --run bench
```

Comment:
nit: On line 34 might make sense to add comment that if command there is updated, it also need to be updated here.

Reply:
Better yet, I need to find a way to define it once use at both places (step and action script).